docroot/themes/custom/yukonca_glider/src/sass/pages/_multiple-step.page.scss

```diff
@@ -0,0 +1,11 @@
+.node--type-multi-step-page {
```

Comment:
We need for general styles for this page

Reply:
Could you please check whether we need this margin in all nodes or now? If this rule is used in different content types then it would be good to make this rule more global.